week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
+    height: 99%;
```

width를 조정하면 그에 맞춰 자동으로 height도 조정됩니다. 일반적으로 height보다는 width를 사용합니다.
```suggestion
    width: 650px;
```